docs/cli.md

```diff
@@ -130,6 +130,20 @@ For instance, for ES6 support with [Babel](https://babeljs.io/) 7 add:
 
 This will effectivally call `require('@babel/register')` prior to requiring any support files.
 
+Note: Your `.babelrc` should target a node environment. For example:
```

Comment:
This is only valid if you are using babel/preset-env. Think that is worth calling out